homeassistant/components/rfxtrx/config_flow.py

```diff
@@ -0,0 +1,22 @@
+"""Config flow for RFXCOM RFXtrx integration."""
+import logging
+
+from homeassistant import config_entries
+
+from . import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for RFXCOM RFXtrx."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_LOCAL_POLL
+
+    async def async_step_import(self, import_config=None):
+        """Handle the initial step."""
+
+        await self.async_set_unique_id(DOMAIN)
+        self._abort_if_unique_id_configured(import_config)
+        return self.async_create_entry(title="RFXTRX", data=import_config)
```

Comment:
We normally check the connection to the device or gateway before creating the entry.

Reply:
While i'd agree on user input. I don't think it make much sense on import since we would have to connect to device twice on every start.

Reply:
We're aborting if there's an existing entry, right? So we won't check twice every start. We'll just check before creating the entry.

Reply:
True. But we would be aborting on a changed entry too, so that wouldn't be checked either. When i add the user flow i'll have to add the code, so will be easy to extend import with the check too then.

Reply:
If the connection details change we should preferably check the connection again before updating the entry.

Reply:
I can't fix this for now. Must get upstream fix first: https://github.com/Danielhiversen/pyRFXtrx/pull/97. Would prefer not to block on this.

Reply:
Ok.